frontend/src/views/SQR/Form.vue

```diff
@@ -471,24 +471,22 @@ export default class Form extends Vue {
     this.debounceSubmitDraft.cancel();
   }
 
-  async submitForm(save = false) {
-    const sqrSubmissionBranch = save
+  async submitForm(submit = false) {
+    const sqrSubmissionBranch = submit
       ? SubmissionsBranch.COMPLETED
       : SubmissionsBranch.DRAFTS;
-    if (!save) {
+    if (!submit) {
       this.draftStatus = FormState.SAVING;
     }
-    if (!save || (this.$refs as any).form.validate()) {
-      if (save) {
-        this.cancelAutoSave();
-        this.draftStatus = FormState.SAVING;
-      }
+    if (!submit || (this.$refs as any).form.validate()) {
       const { created, changed, completed, ...form } = this.form;
       const data: any = {
         ...form,
         changed: firebase.database.ServerValue.TIMESTAMP
       };
-      if (save) {
+      if (submit) {
+        this.cancelAutoSave();
+        this.draftStatus = FormState.SAVING;
```

Comment:
Setting  `draftStatus` inside `submit===true` branch of execution does not seem intuitive. Maybe it's not a draft status but a form status?

Moreover, when we submit a form (not saving a draft) it might be more informative to show something like “Submitting…” instead of “Saving…”